plugins/modules/net_tools/nmcli.py

```diff
@@ -1832,8 +1847,8 @@ def _compare_conn_params(self, conn_info, options):
 
             if key in conn_info:
                 current_value = conn_info[key]
-                if key == 'ipv4.routes' and current_value is not None:
-                    # ipv4.routes do not have same options and show_connection() format
+                if key == 'ipv4.routes' or key == 'ipv6.routes' and current_value is not None:
```

Comment:
You have to write it
```suggestion
                if (key == 'ipv4.routes' or key == 'ipv6.routes') and current_value is not None:
```
otherwise it does something else than expected. It would be more Pythonic to write it as follows, though:
```suggestion
                if key in ('ipv4.routes', 'ipv6.routes') and current_value is not None:
```

Reply:
done